README.md

```diff
@@ -7,7 +7,7 @@ This service can be used with any project type. The examples below are Drupal-sp
 ## Install/Update
 
 1. `ddev get drud/ddev-selenium-standalone-chrome`
-2. Optional. Update the provided .ddev/config.selenium-standalone-chrome.yaml as you see fit(and remove the #ddev-generated line). You can also just override lines in your .ddev/config.yaml
+2. Optional. Update the provided .ddev/config.selenium-standalone-chrome.yaml as you see fit (and remove the #ddev-generated line). You can also just override lines in your .ddev/config.yaml. Note that this installs the seleniarm image by default (for Apple Silicon machines), so if you are on x86 you probably want to use the [docker-selenium images](https://github.com/SeleniumHQ/docker-selenium) instead.
```

Comment:
This is not correct. The image (https://hub.docker.com/r/seleniarm/standalone-chromium/tags?page=1&name=4.1.4-20220429) has both arm64 and amd64 versions, and the correct one is automatically pulled from hub.docker.com based on your architecture. The user does not have to be involved with thinking about architecture here at all.  This is true of all the images DDEV uses, they all have arm64 and amd64 pushes that make this possible.

```suggestion
2. Optional. Update the provided .ddev/config.selenium-standalone-chrome.yaml as you see fit (and remove the #ddev-generated line). You can also just override lines in your .ddev/config.yaml.
```

Reply:
Wow, that's an awesome feature.  I mistakenly added this comment based on a reading of the seleniarm README without understanding how ddev works.